splash/server.py

```diff
@@ -33,6 +33,7 @@ def parse_opts(jupyter=False, argv=None):
         help="path to a folder with proxy profiles")
     op.add_option("--js-profiles-path",
         help="path to a folder with javascript profiles")
+    op.add_option("--maxrss-check-interval", type=int,help="set custom interval for checking rss (in seconds) (default: %default)", default=60)
```

Comment:
Please move it underneath the `maxrss` option so it will be clearer they are connected.